pages/document-upload.tsx

```diff
@@ -0,0 +1,246 @@
+        }
+        setIsUploading(false);
+    };
+
+    const uploadDocumentUI = (): JSX.Element => {
+        return (
+            <Wrapper session={session}>
+                <BackButton />
+                <VStack>
+                    <Center>
+                        <Box width="700px" mb="40px">
+                            <Center>
+                                <Text fontWeight="700" fontSize="36px" m="40px">
+                                    Upload Document
+                                </Text>
+                            </Center>
+                            <Center>
+                                <DragAndDrop setFiles={setFiles} />
+                            </Center>
+                        </Box>
+                    </Center>
+                    {files.map((file: File) => (
+                        <Text key={file.name}>
+                            Document uploaded: <u> {file.name} </u>
+                        </Text>
+                    ))}
+                    <Center paddingBottom="40px">
+                        {!!files.length && !isUploading && (
+                            <Button
+                                backgroundColor="#0C53A0"
+                                borderColor="brand.400"
+                                width="366px"
+                                height="54px"
+                                fontSize="16px"
+                                fontWeight="400"
+                                color="white"
+                                border="1px"
+                                mt="20px"
+                                onClick={upload}
+                            >
+                                Submit
+                            </Button>
+                        )}
+                        {isUploading && (
+                            <Button
+                                backgroundColor="#0C53A0"
+                                borderColor="brand.400"
+                                width="366px"
+                                height="54px"
+                                fontSize="16px"
+                                fontWeight="400"
+                                color="white"
+                                border="1px"
+                                mt="20px"
+                            >
+                                <Spinner
+                                    thickness="4px"
+                                    speed="0.65s"
+                                    emptyColor="gray.200"
+                                    color="blue.500"
+                                    size="xl"
+                                />
+                            </Button>
+                        )}
+                    </Center>
+                </VStack>
+            </Wrapper>
+        );
+    };
+
+    const uploadSuccessUI = (): JSX.Element => {
+        return (
+            <Wrapper session={session}>
+                <BackButton />
```

Comment:
For the success page, we want a close button instead as suggested by our designers.
```suggestion
                <Flex justifyContent="flex-end">
                    <CloseButton />
                </Flex>
```